contrib/vault/roles/vault/defaults/main.yml

```diff
@@ -114,7 +114,7 @@ vault_client_headers:
   Content-Type: "application/json"
 
 etcd_cert_dir: /etc/ssl/etcd/ssl
-kube_cert_dir: /etc/kubernetes/ssl
+kube_cert_dir: /etc/kubernetes/pki
```

Comment:
This is a breaking change.

You would need to create a task to move old certs then or upgrade will fail.